css/style.css

```diff
@@ -62,9 +57,21 @@ ul {
 
 .list__link {
     position: relative;
+    text-transform: uppercase;
+}
+
+.list__link-active {
```

Comment:
Change class name according to BEM convention

Reply:
Class name changed